signingscript/test/test_sign.py

```diff
@@ -947,3 +947,63 @@ def test_signreq_task_keyid():
 
     with pytest.raises(SigningScriptError):
         result = await sign.sign_gpg_with_autograph(context, tmp, 'gpg')
+
+
+# sign_omnija {{{1  -- 537
+@pytest.mark.asyncio
+@pytest.mark.parametrize('filename,raises', (
+    ('foo.unknown', True),
+    ('foo.zip', False),
+    ('foo.dmg', False),
+    ('foo.tar.bz2', False),
+))
+async def test_sign_omnija(context, mocker, filename, raises):
```

Comment:
To cover `if files_to_sign:`, you could add a parameter to override `files` `if not None`. Then if we don't have any files named `omni.ja`, you'll cover that branch.